### DISCUSSION TITLE
I am going to override header.html and delete the icon. Should I be worried about missing references?

### DISCUSSION THREAD

**Discussion by LizThompson93** [squidfunk/mkdocs-material#7088] [Q&A] [answered]
Not sure how integral "md-header__button md-icon" is to the functioning mkdocs/material. A little worried that it might break something, anything I should be aware of?

**Comment by squidfunk** [comment, marked as answer]
Deleting or adding syntax from and to partials should always work without problems. If not, please create an issue ☺️

**Comment by Marlony-Gabriel** [comment]
Hello people. 
I'm about to try an override in header.html and have some questions/"fears".
I need to include a script from jivochat to see if it will work.
I'm trying to override header.html because I'd tried to include in extra.js but the scripts includes are inserted in the final part of the code page. According to jivochat, the script provided to integrate the chat in you site, need to be in the head section. Any ideias or precautions that I need to pay atention before do it ?
Thank you.

**Reply by kamilkrzyskow** [reply]
Hello @Marlony-Gabriel,
if you need to add custom JS in the `<head>` tag, then extending the theme via blocks should be sufficient.
https://squidfunk.github.io/mkdocs-material/customization/#overriding-blocks
```jinja2
{% extends "base.html" %}

{% block extrahead %}
  <script...>
{% endblock %}
```

The added content wil be added here in the final HTML:
https://github.com/squidfunk/mkdocs-material/blob/c1336ae0f598a47828c44073fb6491f622280e0d/src/templates/base.html#L172-L173

**Reply by Marlony-Gabriel** [reply]
Thank you @kamilkrzyskow . I trying to do what you say, but i'm failing in make my custom_dir works.
Bellow is the important part of my mdocs.yml project, wich name is 'duvidas'.

`site_name: Central de Ajuda
site_url: https://centraldeajuda.serpro.gov.br

pages:
nav:
  - Central de Ajuda: 
|
|

theme:
  name: material
  custom_dir: overrides
  logo: imgs/logo-serpro.png
  favicon: imgs/favicon.png
  language: pt-BR
  font:
    text: Open Sans

include_homepage_in_sidebar: false
markdown_extensions:
  attr_list: {}
  admonition: {}
  codehilite: {}
  meta: {}

extra_css:
  - css/extra.css`

So, where  should I create the overrides folder ? Inside my project, or where material theme is installed.
I try to build the project with booth options and got the erros listed bellow.

**With overrides folder inside my project:**
ERROR    -  Config value 'theme': The path set in custom_dir ('C:\Trabalho\Documentação Produtos\duvidas\overrides')
            does not exist.
Aborted with 1 Configuration Errors!

**With overrides folder in material theme folder (C:\Users\S813478121\AppData\Local\Programs\Python\Python310\Lib\site-packages\material\overrides)**
RecursionError: maximum recursion depth exceeded in comparison

**Reply by kamilkrzyskow** [reply]
@Marlony-Gabriel 
You need to create an `overrides` folder next to the `mkdocs.yml` file. All paths, unless said otherwise in the documentation, are relative to the `mkdocs.yml` parent directory. You should never need to change the files of the theme directly in its installation folder.
```python
# /c/MyFiles/_git/removable/overrides-crash
$ mkdocs build
ERROR   -  Config value 'theme': The path set in custom_dir
           ('C:\MyFiles\_git\removable\overrides-crash\overrides') does not exist.

Aborted with a configuration error!
```
```python
# /c/MyFiles/_git/removable/overrides-crash
$ mkdir overrides
```
```python
# /c/MyFiles/_git/removable/overrides-crash
$ mkdocs build
INFO    -  Cleaning site directory
INFO    -  Building documentation to directory: C:\MyFiles\_git\removable\overrides-crash\site
INFO    -  Documentation built in 2.12 seconds
```

**Reply by Marlony-Gabriel** [reply]
Thank you very much @kamilkrzyskow . With your tips I can make the changes works. I extended header.html in place of base.html because base.html keeps given me the same RecursionError. I think it's is because of the size of the project. I have a huge documentation site.
Again, thank you for your time and patience.
